<?xml version="1.0"?>
<launch>

  <node pkg="task_nodes" type="lane_keep_and_stop1" name="lane_keep_and_stop1" output="screen" >
    <remap from="cmd_vel" to="/vehicle/cmd_vel" />
    <param name="stop_time" value="200.0" />
    <param name="sensitve_range" value="4.0" />
    <remap from="stop_line_dist" to="/lane_camera/stop_line_dist" />
    </node>
  
  <include file="$(find task_nodes)/launch/startup/live_lane_navigation.launch" />
  
  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" />
  
  <node pkg="rviz" type="rviz" name="sequoia_nav_laser_rviz" respawn="true" args="-d $(find sequoia_nav_laser)/rviz/rviz_config.rviz" />

</launch>
